### DISCUSSION TITLE
Fine systems?

### DISCUSSION THREAD

**Discussion by eebop** [endless-sky/endless-sky#10013] [Balance, Mechanics, and Interactions]
Hi, I've seen a couple people talking about different ways to implement fine systems.
I have (and I think several others have also) the idea to make it pop up in the talk-to-ship window. Then you could select "Ok" and "Never", or something along those lines. If you select never, you will become hostile. 
Is there a community consensus about fine systems?
Also, is there somewhere with a list of consensuses about things like these?

**Comment by bene-dictator** [comment]
There is a PR that implements some of the ideas discussed here. https://github.com/endless-sky/endless-sky/pull/6376

However, it has been inactive for some time.
